### PR TITLE
Initial Synack API Integration Solution

### PR DESCRIPTION
**Pull request by elforb**
Synack Solution:
- Custom-made createUiDefinition.json
- maintemplate.json file with resources from azuredeploy file given.

Pending:
- Certain URI's in the createUiDefinition.json need be filled in, once provided by Preeti.

### CODE REVIEW

File: Solutions/Synack/SolutionMetadata.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+	"publisherId": "synack",
+	"offerId": "synack",
+    "providers": ["Microsoft"],
+	"categories": {
+		"domains" : ["Security - Others"]
+	},
+	"support": {
+        "tier": "Microsoft",
```
Comment (preetikr) [inline Solutions/Synack/SolutionMetadata.json:9]: tier needs to be partner
support needs to be Synack so please remove Microsoft from support. This is a new pattern of solutions so we are helping by just building it and not publishing.

File: Solutions/Synack/SolutionMetadata.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+	"publisherId": "synack",
+	"offerId": "synack",
+    "providers": ["Microsoft"],
+	"categories": {
+		"domains" : ["Security - Others"]
```
Comment (preetikr) [inline Solutions/Synack/SolutionMetadata.json:6]: I think we need a better category for this

File: Solutions/Synack/Package/mainTemplate.json [hunk 3]
```diff
@@ -0,0 +1,191 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "author": "{{author}}",
```
Comment (preetikr) [inline Solutions/Synack/Package/mainTemplate.json:5]: solution author is Synack

**Review by preetikr — CHANGES_REQUESTED** [review]

**Comment by elforb** [comment]
@preetikr I've updated the metadata values as suggested, in addition to adding the links from the design pattern document. 

Additionally please let me know if any issues with the current Configuration view of the UI:
image

Note: _Due to the way the individual steps work, the various links required are not able to be placed inline within the text._

**Review by NikTripathi — APPROVED** [review]

**Comment by v-dvedak** [comment]
Closing this PR as Preeti has already merged the package.

**Pull request closed without merging**